.github/workflows/commit-lint.yml

```diff
@@ -0,0 +1,14 @@
+name: Lint Commit Messages
+
+on:
+  push
+
+jobs:
+  # see https://github.com/marketplace/actions/commit-linter
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v2
```

Comment:
What would happen if we tried running [`gitlint` via pre-commit](https://jorisroovers.com/gitlint/#using-gitlint-through-pre-commit) instead so we have parity between our pre-commit hooks and CI?

```bash
pre-commit run gitlint --hook-stage commit-msg --commit-msg-filename .git/COMMIT_EDITMSG
```

Reply:
Didn't think of this until now, but gitlint should actually already be running as part of pre-commit ci. (There's no set-up for pre-commit ci; it just uses the same `.yaml` file as local pre-commit.) I suspect it's not working because gitlint requires that extra step of `pre-commit install --hook-type commit-msg`, and I can't think of how to get pre-commit ci to do that.

Reply:
Might be worth considering switching to running pre-commit in a github action instead of using pre-commit ci? Found a couple downsides to pre-commit ci so far:
- if you have to install extra hooks, they don't seem to work
- you have to manually add eslint extensions to `additional_dependencies` every time you extend eslint
- there doesn't seem to be a lot of support or documentation for pre-commit ci

Reply:
> What would happen if we tried running [`gitlint` via pre-commit](https://jorisroovers.com/gitlint/#using-gitlint-through-pre-commit) instead so we have parity between our pre-commit hooks and CI?
> 
> ```shell
> pre-commit run gitlint --hook-stage commit-msg --commit-msg-filename .git/COMMIT_EDITMSG
> ```

I looked into this with commitlint and there's a similar command, but it looks like it only checks the most recent commit message. So if people don't push on every commit (I don't), then commit messages can be skipped